packages/scripts/src/admin-proposals/change-final-fee/2_VerifyRelays.ts

```diff
@@ -28,32 +27,33 @@ import {
   getAddress,
   getContractInstance,
   hre,
-  tokensToUpdateFee,
+  isSupportedNetwork,
+  parseAndValidateTokensConfig,
 } from "./common";
 
 async function main() {
   const shouldForkNetwork = process.env.FORK_NETWORK === "true";
   const callData = process.env["PROPOSAL_DATA"];
   if (!callData) throw new Error("PROPOSAL_DATA environment variable not set");
 
-  if (!process.env.NEW_FINAL_FEE_USD) throw new Error("NEW_FINAL_FEE_USD is not set");
-  if (!process.env.NEW_FINAL_FEE_WETH) throw new Error("NEW_FINAL_FEE_WETH is not set");
-
-  const newFinalFeeUSD = Number(process.env.NEW_FINAL_FEE_USD);
-  const newFinalFeeWeth = Number(process.env.NEW_FINAL_FEE_WETH);
+  const tokensToUpdate = parseAndValidateTokensConfig(process.env.TOKENS_TO_UPDATE);
 
   const { governorRootRelays, governorHubRelays } = decodeRelayMessages(callData);
 
-  const l2NetworksNumber = { polygon: 137, optimism: 10, arbitrum: 42161 };
-
-  for (const [networkName, networkId] of Object.entries(l2NetworksNumber)) {
+  for (const [networkName, networkId] of Object.entries({ polygon: 137, optimism: 10, arbitrum: 42161 })) {
```

Comment:
this should be defined somewhere towards the top so its easy to reference when updating chains or creating envs